src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenByIntegratedWindowsAuthParameterBuilder.cs

```diff
@@ -46,6 +46,18 @@ public AcquireTokenByIntegratedWindowsAuthParameterBuilder WithUsername(string u
             return this;
         }
 
+        /// <summary>
+        /// Enables MSAL to read the federation metada for a wstrust exchange from a file instead of acquiring it from an endpoint.
+        /// See aka.ms/MsalFederationMetadata
+        /// </summary>
+        /// <param name="fileName">Path to federation metada xml file.</param>
+        /// <returns></returns>
```

Comment:
I think that's a new aka.ms link? I think our format is something like: `https://aka.ms/msal-net-federation-metadata`.

```suggestion
        /// <summary>
        /// Enables MSAL to read the federation metadata for a WS-Trust exchange from a file instead of acquiring it from an endpoint.
        /// See https://aka.ms/MsalFederationMetadata
        /// </summary>
        /// <param name="fileName">Path to federation metadata XML file.</param>
        /// <returns></returns>
```